apps/codeforafrica/src/components/Footer/Footer.js

```diff
@@ -23,24 +25,38 @@ const FooterRoot = styled(Box)(
 );
 
 const Footer = React.forwardRef(function Footer(props, ref) {
-  const { subscription, menu } = props;
+  const { subscription, description, footerLinks, socialMedia, menu } = props;
 
   return (
-    <FooterRoot ref={ref}>
+    <FooterRoot component="footer" ref={ref}>
       <Section sx={{ px: { xs: "20px", sm: 0 } }}>
-        <Grid container justifyContent="flex-end">
-          <Grid item xs={12} md={3} sx={{ order: { xs: 0, md: 2 } }}>
-            <NewsletterSubscription {...subscription} />
+        <Grid container spacing={2}>
+          <Grid item xs={12} md={4} sx={{ order: { xs: 2, md: 0 } }}>
+            <Grid container>
+              <Grid
+                sx={{ textAlign: { xs: "center", md: "left" } }}
+                item
+                xs={12}
+              >
+                <FooterDescription description={description} />
+              </Grid>
+              <Grid item xs={12}>
+                <StayInTouch socialMedia={socialMedia} />
+              </Grid>
+            </Grid>
           </Grid>
           <Grid
             item
+            container
+            justifyContent="center"
             xs={12}
-            md={3}
-            sx={{
-              order: { xs: 2, md: 1 },
-            }}
+            md={4}
+            sx={{ order: { xs: 2, md: 1 } }}
           >
-            <FooterNavList menu={menu} />
+            <FooterLinks footerLinks={footerLinks} menu={menu} />
```

Comment:
May be `additionalLinks`, `extraLinks`, etc.? `footerLinks` in `FooterLink` doesn't sound right.